ansible/files/sonic_lab_devices.csv

```diff
@@ -1,5 +1,8 @@
-Hostname,ManagementIp,HwSku,Type
+Hostname,ManagementIp,HwSku,Type,Protocol
 str-msn2700-01,10.251.0.188/23,Mellanox-2700,DevSonic
 str-7260-10,10.251.0.13/23,Arista-7260QX-64,FanoutLeaf
 str-7260-11,10.251.0.234/23,Arista-7260QX-64,FanoutRoot
 str-acs-serv-01,10.251.0.245/23,TestServ,Server
```

Comment:
For devices do not have protocol information, maybe an extra comma at the end of line is required to make these lines conform with the CSV headers? For example:
```
str-msn2700-01,10.251.0.188/23,Mellanox-2700,DevSonic,
str-7260-10,10.251.0.13/23,Arista-7260QX-64,FanoutLeaf,
str-7260-11,10.251.0.234/23,Arista-7260QX-64,FanoutRoot,
str-acs-serv-01,10.251.0.245/23,TestServ,Server,
```